cpp/src/arrow/compute/kernels/hash_aggregate.cc

```diff
@@ -442,6 +442,9 @@ struct GrouperImpl : Grouper {
 };
 
 struct GrouperFastImpl : Grouper {
+  static constexpr int kBitmapPaddingForSIMD = 64;  // bits
+  static constexpr int kPaddingForSIMD = 32;        // bytes
```

Comment:
Could you add a comment explaining why these are 64 and 32 respectively?

Reply:
Unfortunately I have no idea why precisely though values. Perhaps @michalursa can tell.